digitaltwin_client/samples/digitaltwin_sample_device_info/digitaltwin_sample_device_info.c

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-// Implements a sample DeviceInfo DigitalTwin interface.  As the name implies, this interface returns
-// information about the device such as the OS version, amount of storage, etc.
-//
+// Implements a sample DeviceInfo DigitalTwin interface of dtmi:azure:DeviceManagement:DeviceInformation;1.
```

Comment:
Is the JSON representing `dtmi:azure:DeviceManagement:DeviceInformation;1` missing from this directory?

Reply:
Yes.  Per #1529 the DeviceInfo needs a permanent spot we can link to.  Note that we checked the sample JSON into our C SDK because its samples, whereas DeviceInfo is a "system" interface we didn't want to checkin.  I get its inconsistent which is why we have the IOU bug.